QueryCiv3/BiqSections/Prto.cs

```diff
@@ -3,9 +3,174 @@
+        public bool AIExplore               { get => Util.GetFlag(Flags1[4], 3); }
+        public bool AIArmy                  { get => Util.GetFlag(Flags1[4], 4); }
+        public bool AICruiseMissile         { get => Util.GetFlag(Flags1[4], 5); }
+        public bool AIAirBombard            { get => Util.GetFlag(Flags1[4], 6); }
+        public bool AIAirDefense            { get => Util.GetFlag(Flags1[4], 7); }
+
+        public bool AINavalPower            { get => Util.GetFlag(Flags1[5], 0); }
+        public bool AIAirTransport          { get => Util.GetFlag(Flags1[5], 1); }
+        public bool AINavalTransport        { get => Util.GetFlag(Flags1[5], 2); }
+        public bool AINavalCarrier          { get => Util.GetFlag(Flags1[5], 3); }
+        public bool AITerraform             { get => Util.GetFlag(Flags1[5], 4); }
+        public bool AISettle                { get => Util.GetFlag(Flags1[5], 5); }
+        public bool AILeader                { get => Util.GetFlag(Flags1[5], 6); }
+        public bool AITacticalNuke          { get => Util.GetFlag(Flags1[5], 7); }
+
+        public bool AIICBM                  { get => Util.GetFlag(Flags1[6], 0); }
+        public bool AINavalMissileTransport { get => Util.GetFlag(Flags1[6], 1); }
+
+        public PRTORACE AvailableTo; // Binary flag for each civ 0-31
+        private fixed byte Flags2[8];
+        public int Type; // 0: land, 1: sea, 2: air
+        public int OtherStrategy;
+        public int HPBonus;
+
+        private fixed byte Flags3[20];
+        public bool SkipTurn                { get => Util.GetFlag(Flags3[0], 0); }
+        public bool Wait                    { get => Util.GetFlag(Flags3[0], 1); }
+        public bool Fortify                 { get => Util.GetFlag(Flags3[0], 2); }
+        public bool Disband                 { get => Util.GetFlag(Flags3[0], 3); }
+        public bool GoTo                    { get => Util.GetFlag(Flags3[0], 4); }
+        public bool Explore                 { get => Util.GetFlag(Flags3[0], 5); }
+        public bool Sentry                  { get => Util.GetFlag(Flags3[0], 6); }
+
+        public bool Load                    { get => Util.GetFlag(Flags3[4], 0); }
+        public bool Unload                  { get => Util.GetFlag(Flags3[4], 1); }
+        public bool Airlift                 { get => Util.GetFlag(Flags3[4], 2); }
+        public bool Pillage                 { get => Util.GetFlag(Flags3[4], 3); }
+        public bool Bombard                 { get => Util.GetFlag(Flags3[4], 4); }
+        public bool Airdrop                 { get => Util.GetFlag(Flags3[4], 5); }
+        public bool BuildArmy               { get => Util.GetFlag(Flags3[4], 6); }
+        public bool FinishImprovements      { get => Util.GetFlag(Flags3[4], 7); }
+
+        public bool UpgradeUnit             { get => Util.GetFlag(Flags3[5], 0); }
+        public bool Capture                 { get => Util.GetFlag(Flags3[5], 1); }
```

Comment:
Do you mean 4 empty bits instead of 4 empty bytes? So telepad would be GetFlag(Flags3[5], 6)? It might be better if you add the flags in here just because I'm not sure I'd do it right.

Reply:
Oops, yeah, I talk about bytes more frequently than bits and wrote the wrong one.  So GetFlag(Flags3[5], 6) would be correct for Telepad.  Seeing if I can add them now...

Reply:
Well, I followed the guide WildWeazel linked to at https://stackoverflow.com/questions/27567846/how-can-i-check-out-a-github-pull-request-with-git to check out your code, which worked.  But when I pushed, it pushed to a new branch on _this_ repo at https://github.com/C7-Game/Prototype/commits/BiqStructs .  Not altogether surprised it didn't push to your repo, somewhat surprised it pushed to a new branch on this repo.

Going to mark this as resolved, and it should be feasible to either cherry pick my commit there for the cleanup branch, or just open a PR for that branch and the one-commit one-file change.